string event_name
#event name, interesting for filtering

time start_time
time end_time
duration duration